src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,39 @@
+import classes from "./Modal.module.css";
+import ReactDom from "react-dom";
+
+interface ModalProps {
+  show: boolean;
+  close: (params?: any) => any;
+  title: string;
+  children?: React.ReactNode;
+}
```

Comment:
1.  Show is a Boolean. It is convenient to name booleans beginning with `is` for example: `isVisible` or `isShown`
2. why `close` have params if they are not used anywhere? Maybe we can remove params from this function?

Reply:
* thanks for the 1.
* Fixed it.
* When I search for the usage, I see examples like this. So I thought, it was how they do in react. But you are totally right. Removed the params.

Reply:
Resolved ✅